src/App.jsx

```diff
@@ -1,17 +1,19 @@
-import { Routes, Route } from "react-router-dom"
-import Header from "./components/Header.jsx"
-import ArticlesList from './components/ArticlesList.jsx'
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/Header.jsx";
+import ArticlesList from "./components/ArticlesList.jsx";
+import SingleArticle from "./components/SingleArticle.jsx";
 
 function App() {
   return (
-    <div className='App'>
+    <div className="App">
       <Header />
       <Routes>
-        <Route path='/' element={<ArticlesList />}/>
-        
+        <Route path="/" element={<ArticlesList />} />
+        <Route path="/articles" element={<ArticlesList />} />
```

Comment:
Having two paths that render the same component is an interesting choice. I would recommend either rendering a different "home" page on the `path="/"` as an alternative.